portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,25 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Khalil's Portfolio</h1>
+      <p>This is my portfolio. There maybe others like it but this one is mine</p>
+
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+
+      <img src="images/ThatLastCaptainsDinner.jpg" width="500" height="600"/>
+      <h2>Who am I?</h2>
+      <p>I was born in the Dominican Republic back in 2001 and lived there until I was in the second grade when I moved to New York. I lived in and went to school in Queens till I was in Highschool, where I still lived in Queens but went to school in Harlem right at the end of the 3 train line. It was there that I fell in love with Computer Science, I learned how to lead, and I began this chapter of my life</p>     
+
+      <h3>Notable projects</h3>
+      <p>TinkoNaut: 2019 robot for frc robotics team Harlem Knights. Worked on general controls and testing of new technology and later on automatic featurs and pid loop tuning.</p>
+      <p>TinkoKong: 2018 robot for frc robotics team Harlem Knights. Worked on helping newer members of programming learn and programmed use of various utility electronics and sensors.</p> 
+      <p>Classroom aid: A small Classroom aid for Chelsey Roebuck of Elite meant to be used in his classes for the Boys and Girl club. Also counted as the final project for College Engineering</p>
```

Comment:
Looking good, but for a future consideration, consider wrapping these lines so that they can be read without scrolling back and forth. People may have multiple files open at once, so keeping things narrower helps to keep everything legible at once. It's usually standard to keep lines at or under 80 characters!